src/System.IO.Compression/src/System/IO/Compression/Crc32Helper.cs

```diff
@@ -62,22 +63,428 @@ internal static class Crc32Helper
             0x5d681b02u, 0x2a6f2b94u, 0xb40bbe37u, 0xc30c8ea1u, 0x5a05df1bu,
             0x2d02ef8du
         };
+        private static readonly uint[] s_crcTable_1 = new uint[256] {
```

Comment:
Could you add a comment citing how these tables were generated and including a citation for the paper that introduces the algorithm?